### PR TITLE
update to dispatcherless version

### PR DESCRIPTION
**Pull request by JordiSubira**
TODO before merging:
- [x] Update `go.mod` reference to latest for RAINS (at the moment pointing to a branch that should be merged)

This change is [Reviewable](https://reviewable.io/reviews/netsec-ethz/scion-apps/255)

### CODE REVIEW

File: pkg/pan/udp_dial.go [hunk 1]
```diff
@@ -53,25 +56,38 @@ func DialUDP(ctx context.Context, local netip.AddrPort, remote UDPAddr,
...
-	raw, slocal, err := openBaseUDPConn(ctx, local)
+	sn := snet.SCIONNetwork{
+		Topology:    host().sciond,
+		SCMPHandler: scmpHandler{},
+	}
+	conn, err := sn.OpenRaw(ctx, net.UDPAddrFromAddrPort(local))
 	if err != nil {
 		return nil, err
 	}
+	ip, ok := netip.AddrFromSlice((conn.LocalAddr().(*net.UDPAddr).IP))
```
Comment (matzf) [inline pkg/pan/udp_dial.go:67]: `ipport := conn.LocalAddr().(*net.UDPAddr).AddrPort()`

Comment (JordiSubira) [reply pkg/pan/udp_dial.go]: Done.

**Review by juagargi — COMMENTED** [review]
> Reviewed 10 of 13 files at r1, 2 of 2 files at r2.
> *[Reviewable](https://reviewable.io/reviews/netsec-ethz/scion-apps/255)* status: 8 of 19 files reviewed, 1 unresolved discussion / 0 of 1 LGTMs obtained / 0 of 1 approvals obtained

**Review by juagargi — APPROVED** [review]
> Reviewed 11 of 11 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/netsec-ethz/scion-apps/255)* status: all files reviewed, 1 unresolved discussion / 0 of 1 LGTMs obtained / 0 of 1 approvals obtained

**Review by juagargi — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/netsec-ethz/scion-apps/255)* status: all files reviewed, 1 unresolved discussion / 0 of 1 LGTMs obtained / 0 of 1 approvals obtained

**Pull request merged**